recipes/r-plnmodels/meta.yaml

```diff
@@ -0,0 +1,98 @@
+{% set version = '0.10.6' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-plnmodels
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/PLNmodels_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/PLNmodels/PLNmodels_{{ version }}.tar.gz
+  sha256: 99db39b816c7949a6497a2d860953a7479f94047bb8b3395c2acf99fb8f759e6
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: knitr, rmarkdown, testthat, covr, pkgdown, biomformat, phyloseq, spelling
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+
+  host:
+    - r-base
+    - r-mass
+    - r-matrix
+    - r-r6
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-corrplot
+    - r-dplyr
+    - r-ggplot2
+    - r-glassofast
+    - r-gridextra
+    - r-igraph
+    - r-magrittr
+    - r-nloptr >=1.2.0
+    - r-tidyr
+    - libopenblas
```

Comment:
Please change that to `libblas` to allow switching https://conda-forge.org/docs/maintainer/knowledge_base.html#switching-blas-implementation

Reply:
Thank you @dbast for your review and suggestion. I'll do that!